app/model/Raffle.scala

```diff
@@ -10,6 +10,8 @@ sealed trait Raffle {
 
   def name:String
 
+  def people:Set[String]
```

Comment:
We were struggling to implement the different states using types. Our `addPerson` method returns a `Raffle`.  So in order to check whether the `addPerson` method works, we added the `people` method to `Raffle`.

In general, I think our solution of capturing state in the type leads to a lot of code duplication, especially in the tests.  Would it be better to have one `Raffle` type and then an enumeration of `RaffleState`'s?